cache/in-memory/src/lib.rs

```diff
@@ -377,12 +412,20 @@ impl InMemoryCache {
 
     /// Gets a user by ID.
     ///
-    /// This is an O(1) operation.
+    /// This is an O(1) operation. This requires the [`GUILD_MEMBERS`] intent.
+    ///
+    /// [`GUILD_MEMBERS`]: ../twilight_model/gateway/struct.Intents.html#associatedconstant.GUILD_MEMBERS
     pub fn user(&self, user_id: UserId) -> Option<Arc<User>> {
         self.0.users.get(&user_id).map(|r| Arc::clone(&r.0))
     }
 
     /// Gets the voice states within a voice channel.
+    ///
+    /// This requires the [`GUILDS`] intent for the initial set of voice states, and further
+    /// requires the [`GUILD_VOICE_STATES`] intent for updates.
```

Comment:
Maybe all of these need to be reworded, but what is being documented here are the cache methods, not usage of intents, or gateway receiving. What do you think of 
```suggestion
    /// This requires both the [`GUILDS`] and [`GUILD_VOICE_STATES`] intents.
```

Reply:
Hm yeah. I think my idea could be better re-worded to incorporate the idea of it being filled behind the scenes. Not gonna die on the hill though since your idea is much simpler.

Reply:
I'll add a note at the top.

Reply:
Works for me